### ISSUE TITLE
TableView.bindSelected(model: ItemViewModel<in T>) bidirectional?

### ISSUE THREAD

**Issue by jschneider** [edvin/tornadofx#1041]
Calling bindSelected binds the given property to the selection model of the table.

I would like to synchronize the selection of a table with another component. I a m using a ItemViewModel for that case.
Unfortunately the table does *not* update the selection based upon the ItemViewModel.

Is this by design? Could we modify the current implementation? Or add a "bindSelectedBidirectional" method?

**Comment by edvin** [comment]
Yes, this is by design. The reason is simply because `TableView.selectionModel.selectedItemProperty` is read only. What you can quite easily do is `yourProperty.onChange { updateTableSelectionHere }`.
